config/routes.rb

```diff
@@ -23,8 +23,8 @@
   end
 
   scope ":slug", module: "tenants", as: "tenant" do
-    get "/" => "items#index"
-    resources :items
+    get "/" => "projects#index"
+    resources :projects
```

Comment:
I agree that we should do this, but it doesn't have to do with the user story we worked on.